chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -103,7 +103,7 @@
   action :create_if_missing
 end
 
-node.default['brightbox-ruby']['version'] = "2.3"
+node.default['brightbox-ruby']['version'] = "2.4"
```

Comment:
OK, that's an unusual usage of "DRY", and I see how it's confusing.

To me DRYness is this: When you change something, do you have to make that change in more than one place?

The ruby version isn't mentioned elsewhere in the _code_, but one way or another we have chosen what Ruby version we're using, and I assume we want the same version here.